corehq/apps/export/views/new.py

```diff
@@ -228,6 +228,8 @@ def post(self, request, *args, **kwargs):
                 return json_response({
                     'redirect': url,
                 })
+            if request.POST.get("count"):
+                return HttpResponse(url)
```

Comment:
curious why this is an `HttpResponse` and not a `JsonResponse` with 'url' as a key in the returned dict/object?

Reply:
IIRC when I used JsonResponse the success message banner at the top of the page wouldn't show up when you redirect to the returned url (just refreshing the page basically), specifically after bulk deleting, which is why it returns an HttpResponse only when there's a 'count' variable.